### PR TITLE
[ISSUE #174] Getting categories for German and French

### PR DESCRIPTION
**Pull request by sammous**
## Description

Adding a mapping to correctly translate categories in French and German
## Motivation and Context
When parsing the french or german wikipedia dump, the categories list were empty.

ISSUE #174 

## How Has This Been Tested?

Tests for datasets are currently all skipped.
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation, and I have updated it accordingly.

### CODE REVIEW

File: textacy/datasets/wikipedia.py [hunk 1]
```diff
@@ -82,7 +88,7 @@
 re_brackets = re.compile(r'\[\s?\]|\(\s?\)')
 
 re_comments = re.compile('<!--.*?-->', flags=re.UNICODE | re.DOTALL)
-re_categories = re.compile(r'\[\[Category:[^\]\[]*\]\]', flags=re.UNICODE)
+re_categories = re.compile(r'\[\[(%s)[^\]\[]*\]\]' % '|'.join(MAPPING_CAT.values()), flags=re.UNICODE)
```
Comment (bdewilde) [inline textacy/datasets/wikipedia.py]: nit: Could you use `str.format()` instead of the older `%s` form?

**Comment by bdewilde** [comment]
Hey, this looks good to me, but just want to confirm: Does it work as expected on the French and/or German Wikipedias?

File: textacy/datasets/wikipedia.py (hunk 1 above)
Comment (sammous) [reply textacy/datasets/wikipedia.py]: sure

**Comment by sammous** [comment]
yes I dumped successfully the French Corpus. I didn't test in German though, I followed what was said in the issue, my German is ... :see_no_evil:

**Comment by sammous** [comment]
I did your modification and tested for German, all good :+1: 
capture du 2018-03-24 15-55-31

and French:
capture du 2018-03-24 20-33-00

**Comment by bdewilde** [comment]
Yesss, thanks for your help, @sammous !

File: textacy/datasets/wikipedia.py [hunk 2]
```diff
@@ -88,7 +88,8 @@
 re_brackets = re.compile(r'\[\s?\]|\(\s?\)')
 
 re_comments = re.compile('<!--.*?-->', flags=re.UNICODE | re.DOTALL)
-re_categories = re.compile(r'\[\[(%s)[^\]\[]*\]\]' % '|'.join(MAPPING_CAT.values()), flags=re.UNICODE)
+
+re_categories = re.compile(r'\[\[({})[^\]\[]*\]\]'.format('|').join(MAPPING_CAT.values()), flags=re.UNICODE)
```
Comment (bdewilde) [inline textacy/datasets/wikipedia.py]: _Although_: Is that not an extra paren before `.join`? I'll hotfix in master if so.

Comment (sammous) [reply textacy/datasets/wikipedia.py]: yes sorry my bad

Comment (bdewilde) [reply textacy/datasets/wikipedia.py]: no worries 👌

Comment (sammous) [reply textacy/datasets/wikipedia.py]: I was just about to fix it before you merged, you too quick ;)
i'll make sure next time to restart kernel in jupyter 🙈

**Pull request merged**